addons/hr_recruitment/data/hr_recruitment_demo.xml

```diff
@@ -490,59 +490,6 @@
         <field name="res_id" ref="hr_recruitment.hr_candidate_mkt0"/>
     </record>
 
-    <!-- Set the main attachment to avoid automatic sending to the OCR-->
```

Comment:
Yes, those records were added to prevent the OCR, but it was before the candidate module came.
Now, the candidate model has the OCR, not the applicant.
The applicant attachment preview is not working, so these lines were removed.

Reply:
Right, I was just about to ask thePO to create a task for it.

Reply:
I don't understand why we should remove this tbh. If there is no OCR, then you can just attach those from the original record creation; but removing those isn't in the specs so we can avoid it

Reply:
Hello @Bertrand2  :wave: 

I’m having an issue where new attachments don’t show up after refreshing the record because of this lines
(Only on the demo data)
so we should remove these lines, and I’ve created a task to clarify the situation
Task: https://www.odoo.com/odoo/my-tasks/4438455